### DISCUSSION TITLE
It can be used with DigitalOcean App?

### DISCUSSION THREAD

**Discussion by dedonnodev** [appwrite/appwrite#2718] [Q&A]
Hi there,
I am planning to switch from Firebase to Appwrite and would like to know if there is a way to use it with https://www.digitalocean.com/products/app-platform/

I've seen that AppWrite can be used with Docker, and that you can configure variables for various databases.
But I don't know if a database for MariaDB and maybe an s3 for the files is enough to keep everything "persistent". Because every time you deploy with DigitalOcean App, all previous files are deleted. In fact it offers you to buy a database that does not reside where docker is deployed.

So, final question, if I used a MariaDB external to docker, does AppWrite become persistent? Or save files and various configurations locally?

**Comment by stnguyen90** [comment]
A lot of people use DO for Appwrite. Typically, people create an Ubuntu droplet, install docker, and then run the Appwrite install command. 

Feel free to reach out on discord if you have any other questions.

**Comment by adityaoberai** [comment]
@3lB4rt0 you would end up setting up Appwrite on a Droplet instead of the App Platform, as @stnguyen90 also mentioned. That way, the database remains persistent.

To speed things up further, you could use the 1-Click App from the DigitalOcean marketplace as well: https://marketplace.digitalocean.com/apps/appwrite (the $10 VM option should be a good starting point for testing).

**Comment by dedonnodev** [comment]
Really thank you for you answers.
When i will start this new project, i will come back and maybe also contribute 👍
Have a good day
